packages/atomic/src/components/commerce/search-box-suggestions/atomic-commerce-search-box-query-suggestions/atomic-commerce-search-box-query-suggestions.tsx

```diff
@@ -51,7 +51,7 @@ export class AtomicCommerceSearchBoxQuerySuggestions {
    */
   @Prop({reflect: true}) public maxWithoutQuery?: number;
 
-  componentWillLoad() {
+  componentDidLoad() {
```

Comment:
Aux4: lifecyle/racing issue around dispatchSearchBoxSuggestionsEvent.
Test without this line.